packages/core/src/packager.ts

```diff
@@ -1,5 +1,11 @@
 // This is a collection of flags that convey what kind of build you want. They
 // are intended to be generic across Packagers, and it's up to Packager authors
+
+export interface FingerprintOptions {
+  enabled?: boolean;
+  exclude?: any[];
```

Comment:
I think its actually:

```suggestion
  exclude?: string[];
```

Not that that is a big deal though...

Reply:
done